src/alliteratedActors.js

```diff
@@ -21,8 +21,14 @@
  *
  * @return {string}
  */
-function alliteratedActors(names) {
-  // write code here
+function alliteratedActors(people) {
+  const alliteratedName = people.find(person => {
+    const fullName = person.split(' ');
+    const firsLetter = fullName[0][0];
+    return fullName.every(name => name[0][0] === firsLetter);
```

Comment:
person is string -> fullName is array of strings -> name is string -> name[0] is first letter of word
you do not need `name[0][0]` just `name[0]`